src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -397,6 +402,17 @@ export class StudyViewPageStore {
         }
     }
 
+    private _isResourceTabOpen = observable.map<boolean>();
```

Comment:
this sounds like boolean when it's map of boolean.  how about openResourceTabMap?